doc/examples/realtime_servo/realtime_servo_tutorial.rst

```diff
@@ -51,6 +51,23 @@ Expected Output
 
 Note that the controller overlay here is just for demonstration purposes and is not actually included
 
+Introspection
+-------------
+
+Here are some tips for inspecting and/or debugging the system.
+
+#. View the :code:`ros2_controllers` that are currently active with :code:`ros2 control list_controllers`. You will see a `JointTrajectoryController <https://github.com/ros-controls/ros2_controllers/tree/master/joint_trajectory_controller>`_ that receives the joint position commands from Servo and handles them in the simulated robot driver. The JointTrajectoryController is very flexible; it can handle any combination of position/velocity/(position-and-velocity) input. Servo is also compatible with `JointGroupPosition <https://github.com/ros-controls/ros2_controllers/tree/master/position_controllers>`_ or `JointGroupVelocity <https://github.com/ros-controls/ros2_controllers/tree/master/velocity_controllers>`_-type controllers.
```

Comment:
The command `ros2 control list controllers` requires `ros2_control` to be installed.  Can you add a note that mentions this, or possibly add it as a dependency in `package.xml`?

Reply:
good point, done